### PR TITLE
Fix duplicate dimension name

### PR DESCRIPTION
**Pull request by ruthmd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -281,23 +291,22 @@ view: headquater_building_floors {
...
 ```LookML
-include: "restaurant_chain.view"
+include: "restaurant_chain.view.lkml"
    
-explore: chains_table {
-  view_name: chains_table
-  from: chains_table
-  label: "chains_table explore"
-  description: "chains_table explore"
+explore: JSON_TABLE {
```
Comment (thoean) [inline README.md]: Can this view in the example be created with table name `chains_table` to keep the diff to a minimum? It allows to easier review / see the outcome and impact of the changes.

File: README.md [hunk 2]
```diff
@@ -156,79 +149,96 @@ LATERAL FLATTEN(OUTER => TRUE, INPUT => chains_table."raw_data_column":"restaura
...
+view: JSON_TABLE { 
+  sql_table_name: RESTAURANT_DETAILS ;;
 
-
-view: chains_table { 
-  sql_table_name: chains_table ;;
-
-  dimension: city {
-    description: "City"
+  dimension: country {
```
Comment (thoean) [inline README.md]: What's the sorting that's applied? Can we create consistently the same result with the same input? Either by keeping the sorting in the JSON (probably the most logical from a user's perspective) or by alphabethical sorting? It's interesting that every commit creates such huge differences.

File: README.md [hunk 3]
```diff
@@ -115,38 +108,38 @@ With J2V all the structures are discovered automatically and two files are gener
...
+,IFNULL(JSON_TABLE."DATA":"version"::string,'N/A') AS VERSION
+,JSON_TABLE."DATA":"dataGenerationTimestamp"::timestamp AS DATAGENERATIONTIMESTAMP
 ,
 ---restaurants Information
-restaurants.VALUE:"address"::string AS RESTAURANTS_ADDRESS
-,restaurants.VALUE:"city"::string AS RESTAURANTS_CITY
-,restaurants.VALUE:"country"::string AS RESTAURANTS_COUNTRY
-,restaurants.VALUE:"currency"::string AS RESTAURANTS_CURRENCY
-,restaurants.VALUE:"name"::string AS RESTAURANTS_NAME
+IFNULL(restaurants.VALUE:"address"::string,'N/A') AS RESTAURANTS_ADDRESS
```
Comment (thoean) [inline README.md]: Was this example created with the IFNULL flag? Should it be created without it to be consistent with the previous example? Or if we want to change the example's output, can we do it in a separate pull request?

File: setup.py [hunk 4]
```diff
@@ -8,9 +8,9 @@
 with open("README.md", "r") as fh:
     setup(
         name='j2v',
-        version='1.3.7',
-        author="Michal Zasadzinski",
-        author_email="michal.zasadzinski@gmail.com",
+        version='1.3.8',
+        author="Michal Zasadzinski, Ruthwik Ganesh",
```
Comment (thoean) [inline setup.py]: Should this be changed to `Cimpress Technology`? Over time, this will be a long list.

File: README.md (hunk 2 above)
Comment (ruthmd) [reply README.md]: Dimensions were initially stored in an unordered set, I'll sort the dimensions based on their names.

File: README.md (hunk 1 above)
Comment (ruthmd) [reply README.md]: Sure I'll make the changes!

**Comment by mzas** [comment]
Changes suggested by @thoean were implemented.
